### ISSUE TITLE
Should the voter and admin codes be configurable?

### ISSUE THREAD

**Issue by prufrock** [hubie/majority_finder#3]
From what I can tell it looks like the voter and admin codes should be configurable or be loaded from a database. What do you think of moving them into one of those places?

https://github.com/hubie/majority_finder/blob/bb87bf66f46f30756beb0420e23dcc51ec4f345d/lib/majority_finder/login/form.ex#L3-L4

**Comment by hubie** [comment]
Yup!  In the short-term I'm planning on either moving these to either a config file or environment variables.  Longer term I'm thinking an admin will upload a CSV-ish file of valid codes that would be generated by the ticketing software.  (Or perhaps the converse where the app generates the CSV of codes that is then input into the ticketing software.)
